app/views/view-claim.njk

```diff
@@ -34,8 +34,8 @@
         }) }}
       {% endif %}
 
-      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number: {{reference}}</h1>
-      <h1 class="govuk-heading-l govuk-!-margin-bottom-7">{{ title }} <span class="govuk-tag {{ status.tagClass }}">{{ status.capitalisedtype }}</span></h1>
+      <h1 class="govuk-caption-l govuk-!-margin-bottom-3">Claim number by <a href="/view-agreement/{{applicationReference}}">{{ title }}</a> </h1>
```

Comment:
It should be "Claim by" not "Claim number by"

Also I think the URL is wrong. Do you have endemics turned ON in your .env file? I think Elamin had this same issue and that fixed it. The new url is /agreement/reference/claims

Reply:
but reference is Claim reference will that work for agreement ?